Source/APNGKit/APNGDecoder.swift

```diff
@@ -29,6 +29,7 @@ class APNGDecoder {
     let reader: Reader
     let options: APNGImage.DecodingOptions
     let cachePolicy: APNGImage.CachePolicy
+    let usesPrimitiveTrueColorAlphaChannel: Bool
```

Comment:
@noppefoxwolf 

Do we actually need this here? Seems this property is only used in `APNGImageRenderer` when creating the bitmap info, so maybe just reading the `options` in the renderer class can be better (so we can eliminate the property)?

Reply:
Thank you for your review.
I moved this parameter to `APNGImageRenderer` and `APNGImageView`.

Reply:
Thanks for it! I will merge it and then do some smaller adjustments to make the APIs more generic.